Meridian59/Meridian59.csproj

```diff
@@ -561,7 +565,9 @@
     <Content Include="zlib\license.txt" />
     <None Include="Resources\Glow.png" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="Protocol\GameMessages\AdminMode\" />
```

Comment:
I don't think this folder "AdminMode" should be there.

AFAIK the network protocol only knows "LoginMode" - the messages with "AP_" prefix from proto.h and "GameMode", the messages with "BP_" prefix from proto.h - or am I missing something?